docs/governance/rfc/006-es-validation.md

```diff
@@ -0,0 +1,16 @@
+# 006: Request validation for EcmaScript
+
+This RFC proposes adding an ancillary EcmaScript project to the Connect Github organization:
+
+* `@connectrpc/validate`, which provides request validation for Connect-ES services using protovalidate.
```

Comment:
nit: should we split out the repo & the intended NPM package coordinate, given that above we're talking about "adding an ancillary EcmaScript project to the Connect Github organization"?

```suggestion
* repository `connectrpc/validate-es` (published as `@connectrpc/es` on NPM), which provides request validation for Connect-ES services using protovalidate.
```

Reply:
In the [first one](https://connectrpc.com/docs/governance/rfc/go-cors-authn) we talked about language packages, so I also used language package name.